components/TestLibraryStepper/TestLibraryStepper.tsx

```diff
@@ -1,41 +1,46 @@
 ﻿import React from "react";
 import {Step, StepLabel, Stepper, Typography} from '@material-ui/core';
 import {MuiThemeProvider} from '@material-ui/core/styles';
-import {CandidateTestStatus} from "../../pages/api/candidateApiClient.module";
 import TestSwitchStepIcon from "../TestLibraryStepperIcons/TestLibraryStepperIcons";
 import {h1Style, TestSwitchConnector, TestSwitchTheme} from "../TestLibraryOverrides/TestLibraryOverrides"
 import Link from "next/link";
 import scss from '../TestLibraryStepper/TestLibraryStepper.module.scss';
 
+interface CandidateTestStatus {
+    testName: string;
+    testStatus: string;
+}
 
 interface TestLibraryStepperProps {
-    candidateTestStatus: CandidateTestStatus[];
+    candidateTestStatuses: CandidateTestStatus[];
 }
 
-function getSteps() {
-    //TODO: mock data, set number of steps based on test range, maybe props.candidateTestStatus.tests.length?
-    const testNumber = 3;
-
+function getSteps(testArr: CandidateTestStatus[]) {
     //set labels for steps
     const testLabelArray = [];
-    while (testLabelArray.length + 1 <= testNumber) {
-        testLabelArray.push(`Test ${testLabelArray.length + 1}`);
-    }
+    testLabelArray.push(testArr.map(test => test.testName));
     return testLabelArray;
 }
```

Comment:
I think this whole method can now just be 
```ts
return testArray.map(test => test.testName);
```

Reply:
of course. done! ✔️